app/Http/Controllers/admin/AdminController.php

```diff
@@ -310,4 +312,27 @@ public static function applicantBaseQuery($query)
 
       return $query;
   }
+
+  public function resendRecEmail()
+  {
+      // get rec info
+      $rec_id = Input::get('rec_id');
+      $recommendation = Recommendation::whereId($rec_id)->firstOrFail();
+      $token = RecommendationToken::where('recommendation_id', $recommendation->id);
+      $link = link_to_route('recommendation.edit', 'Please provide a recommendation', [$recommendation->id, 'token' => $token]);
+
+      // get applicant info
+      $applicant_id = Input::get('applicant_id');
+      $applicant = User::whereId($applicant_id)->firstOrFail();
+
+      // build and send email
+      $email = new Email();
+      $data = [
+      'link'           => $link,
```

Comment:
okay I tabbed in the items in the array, but do we want the => to align with each other or no?